bubble-sort.rb

```diff
@@ -0,0 +1,30 @@
+
+def bubble_sort(array)
+  for i in 0..array.size - 2
+    for j in 0..array.size - 2
+      if (array[j] > array[j + 1])
+        tmp = array[j]
+        array[j] = array[j + 1]
+        array[j + 1] = tmp
+      end
+    end
+  end
+
+  return array
+end
+
+def bubble_sort_by(array)
```

Comment:
this method isn't working without black, kindly check for whether the block is given or not. Also in the project, it's required to use spaceship operator for comparison.

Reply:
Please about *spaceship*, it's instead said that 

> The block’s return will be similar to the spaceship operator you learned about before: If the result of the block execution is negative, the element on the left is “smaller” than the element on the right. 0 means both elements are equal. A positive result means the left element is greater

It said `Similar` not that with have to use it

Reply:
Sorry, my bad. Thanks for bringing that up 😊
Although, you need to fix it for a block.